domains/mikasa.is-a.dev.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "",
+        "email": "mohitshedge5@gmail.com"
+    },
+    "record": {
+        "CNAME": "superb-florentine-e0f2ef.netlify.app"
```

Comment:
we dont support Netlify at the moment (though we will soon)

Reply:
Netlify links are unsupported. (Also, the website seems incomplete?)